can/interfaces/pcan/PCANBasic.py

```diff
@@ -661,9 +611,6 @@ def GetValue(
             logger.error("Exception on PCANBasic.GetValue")
             raise            
 
-    # Returns a descriptive text of a given TPCANStatus
-    # error code, in any desired language
```

Comment:
How will the developer know this without the comment? I suggest either reverting or moving some of the comments into proper python docstrings.

Reply:
Actually, it already is in the doc string. ;-)